openshift/recovery-controller/src/main/java/me/snowdrop/boot/narayana/openshift/recovery/StatefulsetRecoveryControllerProperties.java

```diff
@@ -60,7 +60,7 @@
     /**
      * Enables log-scraping based error detection during recovery.
      */
-    private boolean logScrapingErrorDetectionEnabled = true;
+    private boolean logScrapingErrorDetectionEnabled = false;
```

Comment:
This changes the default value of the bean, but I think the `LogScrapingRecoveryErrorDetector` is also bound to the registry because spring-boot does not read this property when instantiating it. See the other comment.